.github/workflows/main.yml

```diff
@@ -0,0 +1,32 @@
+on:
+  workflow_dispatch:
+  schedule:
+  - cron: "0 2 * * 5"
+jobs:
+  update-symbols:
+    permissions:
+      contents: write
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+    steps:
+    - uses: actions/checkout@v2
+    - name: Set up Python
+      uses: actions/setup-python@v2
```

Comment:
You will find that the versions specified here will trigger warnings in the CI summary related to using an old version of node.js. The fix for this is to use the latest version:
```diff
-    - uses: actions/checkout@v2
+    - uses: actions/checkout@v3
     - name: Set up Python
-      uses: actions/setup-python@v2
+      uses: actions/setup-python@v4
```
Sorry, I'm a little late to give this insight.

Reply:
done